src/components/app/constants.js

```diff
@@ -73,3 +73,12 @@ export const FOOTER_CLASSNAMES = {
     DYNAMIC_FOOTER: 'ant-col-xs-24 ant-col-sm-24 ant-col-md-6',
     ROW: 'ant-row',
 };
+
+export const VALID_SIGNOUT = "Valid sign out";
+
+export const SIGNIN_MESSAGES = {
+    FAILURE: "email or password incorrect",
```

Comment:
Grab your messages from returned data after sigin

Reply:
You mean signin right? OK